server/src/middlewares/ensureAuthenticated.test.ts

```diff
@@ -2,6 +2,21 @@ import { Request, Response } from 'express';
 import ensureAuthenticated from './ensureAuthenticated';
 
 describe('ensures', () => {
+  // FIXME not actually throwing WTF but when env is actually no set on server it throws. I don't think it have anything to do with asynchronous tests and such, since they are on different threads.
```

Comment:
I guess it won't throw unless the JWT token is missing. Invalid tokens, etc are caught by our try-catch; only missing tokens throw errors (`Error`).
Is `authConfig.jwt.secret` set and loaded correctly? I don't think it is related but as a last resort...

Reply:
Not the JWT token, the JWT_SECRET env